internal/controller/logger_manager.go

```diff
@@ -27,23 +27,26 @@ type ClientConnInterface interface {
 // BufferedGrpcWriteSyncer is a custom zap writesync that writes to a grpc stream
 // In case stream is not connected it will buffer to memory
 type BufferedGrpcWriteSyncer struct {
-	client   pb.KubernetesInfoService_SendLogsClient
-	conn     ClientConnInterface
-	buffer   []*zapcore.Entry
-	mutex    sync.Mutex
-	done     chan struct{}
-	logger   *zap.SugaredLogger
-	logLevel zap.AtomicLevel
-	encoder  zapcore.Encoder
+	client         pb.KubernetesInfoService_SendLogsClient
+	conn           ClientConnInterface
+	buffer         []*zapcore.Entry
+	mutex          sync.Mutex
+	done           chan struct{}
+	logger         *zap.SugaredLogger
+	logLevel       zap.AtomicLevel
+	encoder        zapcore.Encoder
+	lostLogEntries int
+	lostLogsErr    error
```

Comment:
Nit. Name these fields consistently.
```suggestion
	lostLogEntriesCount int
	lostLogEntriesErr   error
```